app/bin/tools/backfill_package_fields.dart

```diff
@@ -47,10 +47,12 @@ Future main(List<String> args) async {
 }
 
 Future<void> _backfillPackageFields(Package p) async {
+  final shouldSetUnlisted = p.isUnlisted != true && p.doNotAdvertise == true;
```

Comment:
I don't think we should shoehorn correction for the previous mistake into script..